qbft/proposal.go

```diff
@@ -1,9 +1,9 @@
 package qbft
 
 import (
-	"bytes"
-	"github.com/bloxapp/ssv-spec/types"
 	"github.com/pkg/errors"
+
+	"github.com/bloxapp/ssv-spec/types"
 )
 
 func (i *Instance) uponProposal(signedProposal *SignedMessage, proposeMsgContainer *MsgContainer) error {
```

Comment:
proposal should include the full input. Proposal justifications can be signed msg headers